src/browsers.ts

```diff
@@ -43,15 +43,22 @@ export async function expand(names: string[]): Promise<wd.Capabilities[]> {
     names = [];
   }
 
-  const unsupported = difference(names, supported());
+  // The supported and detect functions are stubbed out in test by
+  // web-component-tester so we need to access them locally this way.
+  const stubbableSupported: typeof supported = module.exports.supported;
```

Comment:
I didn't think so, but it turns out that yeah, this works, typescript gets this right and rewrites local references such that they're externally assignable.